### DISCUSSION TITLE
VS Code button, hooray! Perfect for PreTeXt...

### DISCUSSION THREAD

**Discussion by StevenClontz** [sagemathinc/cocalc#6185] [Ideas]
I primarily use CoCalc via VS Code these days, so I really dig the easy access to VS Code I noticed in the toolbar today. Thanks for implementing that!

Over at https://github.com/PreTeXtBook/pretext-cli we're eyeing VS Code as the IDE of choice for authoring, and have a VS Code plugin that eventually will remove the need for using the terminal for standard tasks. (All @OscarLevin) Any chance. when it's ready, that we could get this plugin installed by default on CoCalc projects (or have a very easy path for free trial users to do so)? I'm seeing a very near future where folks can trial authoring and building in PreTeXt from their browser in just a few clicks without touching a terminal (and I expect many will grab a CoCalc subscription so all my projects can live in the cloud without having to sync several devices).

**Comment by williamstein** [comment]
Yes of course!

On Thu, Oct 20, 2022 at 9:34 AM Steven Clontz ***@***.***>
wrote:

> I primarily use CoCalc via VS Code these days, so I really dig the easy
> access to VS Code I noticed in the toolbar today. Thanks for implementing
> that!
>
> Over at https://github.com/PreTeXtBook/pretext-cli we're eyeing VS Code
> as the IDE of choice for authoring, and have a VS Code plugin that
> eventually will remove the need for using the terminal for standard tasks.
> (All @oscarlevin <https://github.com/oscarlevin>) Any chance. when it's
> ready, that we could get this plugin installed by default on CoCalc
> projects (or have a very easy path for free trial users to do so)? I'm
> seeing a very near future where folks can trial authoring and building in
> PreTeXt from their browser in just a few clicks without touching a terminal
> (and I expect many will grab a CoCalc subscription so all my projects can
> live in the cloud without having to sync several devices).
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/sagemathinc/cocalc/discussions/6185>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAJXS5R7DAJBQNG4VE3XJH3WEFYCXANCNFSM6AAAAAARKKWGIY>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>
-- 
-- William Stein

**Reply by StevenClontz** [reply]
Thanks! We'll be in touch...
